app-misc/sl/sl-5.02.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="sophisticated graphical program which corrects your miss typing"
+HOMEPAGE="http://www.tkl.iis.u-tokyo.ac.jp/~toyoda/index_e.html https://github.com/mtoyoda/sl/"
+SRC_URI="https://github.com/mtoyoda/${PN}/archive/${PV}.tar.gz"
+
+LICENSE="freedist"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~x86-interix ~amd64-linux ~x86-linux ~ppc-macos"
+
+DEPEND="sys-libs/ncurses:0"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	sed -e "s/-lncurses/\`pkg-config --libs ncurses\`/" -i Makefile
+	eapply_user
```

Comment:
It probably makes sense to apply user patches before the sed, as sed can change context and make patches made against upstream source no longer apply.